docs/resources/group.md

```diff
@@ -38,7 +38,7 @@ resource "twingate_group" "aws" {
 
 ### Read-Only
 
-- `id` (String) Autogenerated ID of the Resource, encoded in base64
+- `id` (String) The ID of the Group
```

Comment:
We should change this to "The ID of the Group resource" since this is an output. This is the Terraform ID, correct?